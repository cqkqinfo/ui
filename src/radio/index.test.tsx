import React from 'react';
import Radio from './index';
import { render } from '@testing-library/react';

describe('radio', () => {
  it('渲染', () => {
    const { baseElement } = render(
      <Radio.Group>
        {[
          { value: '1', label: '西瓜' },
          { value: '2', label: '香蕉' },
          { value: '3', label: '苹果' },
          { value: '4', label: '桔子' },
          { value: '5', label: '芒果' },
          { value: '6', label: '甘蔗' },
          { value: '7', label: '火龙果' },
        ].map(item => (
          <Radio value={item.value} key={item.value}>
            {item.label}
          </Radio>
        ))}
      </Radio.Group>,
    );

    expect(baseElement).toMatchSnapshot();
  });
});
