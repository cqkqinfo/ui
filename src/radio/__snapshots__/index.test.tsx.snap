// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`radio 渲染 1`] = `
<body>
  <div>
    <div
      class="group"
    >
      <div
        class="container radio"
        style="background: none;"
      >
        <div
          class="dot"
          style="border-color: #eee; border-width: 1px; background: rgb(255, 255, 255);"
        />
        西瓜
      </div>
      <div
        class="container radio"
        style="background: none;"
      >
        <div
          class="dot"
          style="border-color: #eee; border-width: 1px; background: rgb(255, 255, 255);"
        />
        香蕉
      </div>
      <div
        class="container radio"
        style="background: none;"
      >
        <div
          class="dot"
          style="border-color: #eee; border-width: 1px; background: rgb(255, 255, 255);"
        />
        苹果
      </div>
      <div
        class="container radio"
        style="background: none;"
      >
        <div
          class="dot"
          style="border-color: #eee; border-width: 1px; background: rgb(255, 255, 255);"
        />
        桔子
      </div>
      <div
        class="container radio"
        style="background: none;"
      >
        <div
          class="dot"
          style="border-color: #eee; border-width: 1px; background: rgb(255, 255, 255);"
        />
        芒果
      </div>
      <div
        class="container radio"
        style="background: none;"
      >
        <div
          class="dot"
          style="border-color: #eee; border-width: 1px; background: rgb(255, 255, 255);"
        />
        甘蔗
      </div>
      <div
        class="container radio"
        style="background: none;"
      >
        <div
          class="dot"
          style="border-color: #eee; border-width: 1px; background: rgb(255, 255, 255);"
        />
        火龙果
      </div>
    </div>
  </div>
</body>
`;
